conf/deployment/vsphere/upi_1az_rhcos_vsan_3m_3w_provider_client.yaml

```diff
@@ -0,0 +1,15 @@
+---
+# This config is suppose to work on DC-CP, DC-ECO, DC-PS
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  enable_hw_virtualization: true
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 3
+  master_replicas: 3
```

Comment:
client doe't have masters right?
Do we still need to include them in config because of some compatibility?
Same for another lines with `master_` prefix.

Reply:
this is Provider configuration as a base for deploying both provider and kubevirt clients. Client would not have config file.